lib/exercism/user.rb

```diff
@@ -110,4 +134,23 @@ def items_where(table, condition)
       problems[result["track_id"]] << result["slug"]
     end
   end
+
+  def exercises_list_sql
```

Comment:
Let's call this `suggested_exercises_sql` or `five_a_day_exercises_sql` so it's clear which exercises we're talking about.